config.json.sample

```diff
@@ -7,5 +7,14 @@
     "url": "https://demo.owncloud.com/index.php/apps/oauth2/api/v1/token",
     "authUrl": "https://demo.owncloud.com/index.php/apps/oauth2/authorize"
   },
-  "apps" : ["files"]
-}
\ No newline at end of file
+  "apps" : ["files"],
+  "menu": {
```

Comment:
Please see https://github.com/owncloud/phoenix/issues/2573 - there is already a configuration setup proposed.

I'd like to follow the defined spec - if you disagree - please let us discuss this in https://github.com/owncloud/phoenix/issues/2573 first.